server/src/main/java/io/spine/server/entity/storage/Methods.java

```diff
@@ -0,0 +1,195 @@
+import static com.google.gson.internal.Primitives.wrap;
+import static io.spine.util.Exceptions.newIllegalStateException;
+import static java.lang.String.format;
+import static java.lang.reflect.Modifier.isPublic;
+import static java.lang.reflect.Modifier.isStatic;
+
+/**
+ * Utilities for working with methods.
+ *
+ * @author Dmytro Dashenkov
+ * @author Alexander Yevsyukov
+ */
+class Methods {
+
+    private static final String GETTER_PREFIX_REGEX = "(get)|(is)";
+    private static final Pattern GETTER_PREFIX_PATTERN = Pattern.compile(GETTER_PREFIX_REGEX);
+    private static final ImmutableSet<String> NULLABLE_ANNOTATION_SIMPLE_NAMES =
+            ImmutableSet.of("CheckForNull", "Nullable", "NullableDecl", "NullableType");
+
+    /** Prevents instantiation of this utility class. */
+    private Methods() {
+    }
+
+    static Method retrieveAnnotatedVersion(Method getter) {
+        Optional<Method> optionalMethod = getAnnotatedVersion(getter);
+        if (!optionalMethod.isPresent()) {
+            throw newIllegalStateException("Method `%s` is not an entity column getter.", getter);
+        }
+        return optionalMethod.get();
+    }
+
+    static Optional<String> nameFromAnnotation(Method getter) {
+        final String trimmedName = getter.getAnnotation(Column.class)
+                                         .name()
+                                         .trim();
+        return trimmedName.isEmpty()
+               ? Optional.absent()
+               : Optional.of(trimmedName);
+    }
+
+    static String nameFromGetter(Method getter) {
+        Matcher prefixMatcher = GETTER_PREFIX_PATTERN.matcher(getter.getName());
+        String nameWithoutPrefix = prefixMatcher.replaceFirst("");
+        String result =
+                Character.toLowerCase(nameWithoutPrefix.charAt(0)) + nameWithoutPrefix.substring(1);
+        return result;
+    }
+
+    static boolean hasNullableReturn(Method getter) {
+        AnnotatedType type = getter.getAnnotatedReturnType();
+        boolean result = isNullable(type.getAnnotations());
+        return result;
+    }
+
+    private static boolean isNullable(Annotation[] annotations) {
+        for (Annotation annotation : annotations) {
+            String simpleName = annotation.annotationType()
+                                          .getSimpleName();
+            if (NULLABLE_ANNOTATION_SIMPLE_NAMES.contains(simpleName)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    static void checkGetter(Method getter) {
+        checkNotNull(getter);
+        checkArgument(GETTER_PREFIX_PATTERN.matcher(getter.getName())
+                                           .find()
+                              && getter.getParameterTypes().length == 0,
+                      "Method `%s` is not a getter.", getter);
+        checkArgument(getAnnotatedVersion(getter).isPresent(),
+                      format("Entity column getter should be annotated with `%s`.",
```

Comment:
Please remove redundant `format`, as `checkArgument` is capable of string formatting on its own.